pkg/types/aws/validation/platform.go

```diff
@@ -49,7 +50,10 @@ func validateUserTags(tags map[string]string, propagatingTags bool, fldPath *fie
 		return allErrs
 	}
 	if len(tags) > 8 {
-		allErrs = append(allErrs, field.Invalid(fldPath, len(tags), "number of user tags cannot be more than 8"))
+		logrus.Warnf("bootstrap s3 object tags will be random ordered as number of user  tags is more than 8")
```

Comment:
In terraform, maps are lexicographically ordered, so I believe the order would not be random...

```suggestion
		logrus.Warnf("Due to a limit of 10 tags on S3 Bucket Objects, only the first eight lexicographically sorted tags will be applied to the bootstrap ignition object, which is a temporary resource only used during installation")
```

Also, log messages--unlike error messages--should be capitalized.